### PR TITLE
Add support for TwoGigabit & TwentyFiveGig interfaces to canonical_map.py

### PR DESCRIPTION
**Pull request by bbmin7b5**
- added support for TwoGigabitEthernet and TwentyFiveGigabitE interfaces found in Cisco 9300 and 9400 series switches.

### CODE REVIEW

File: napalm/base/canonical_map.py [hunk 1]
```diff
@@ -99,6 +101,8 @@
     "Serial": "Se",
     "TenGigabitEthernet": "Te",
     "Tunnel": "Tu",
+    "TwoGigabitEthernet": "Tw",
+    "TwentyFiveGigE": "Twe",
```
Comment (ktbyers) [inline napalm/base/canonical_map.py]: @bbmin7b5 Wouldn't it be better to be "Two" and "Twe" as "Tw" could be ambiguous with both "TwoGigabitEthernet" and "TwentyFiveGigE"?

File: napalm/base/canonical_map.py [hunk 2]
```diff
@@ -72,6 +72,8 @@
     "Tunnel": "Tunnel",
     "Tun": "Tunnel",
     "Tu": "Tunnel",
+    "Tw": "TwoGigabitEthernet",
+    "Twe": "TwentyFiveGigE",
```
Comment (ktbyers) [inline napalm/base/canonical_map.py]: @bbmin7b5 Wouldn't it be better to be "Two" and "Twe" as "Tw" could be ambiguous with both "TwoGigabitEthernet" and "TwentyFiveGigE"?

File: napalm/base/canonical_map.py (hunk 1 above)
Comment (bbmin7b5) [reply napalm/base/canonical_map.py]: @ktbyers I went with "Tw" for the "TwoGigabitEthernet" as that is how Cisco displayed the shortened form of the interface when using "show interface status" and "show ip interface brief". I figured I'd follow that model as that is what the other key/value pairs seemed to conform to.

I hope my reasoning makes sense.

**Comment by ktbyers** [comment]
Made minor update here as I was pretty convinced the reverse map should not have any collisions in it (i.e. should be unique as far as abbreviated names):

https://github.com/napalm-automation/napalm/pull/1107

**Comment by bbmin7b5** [comment]
Thanks @ktbyers

**Pull request closed without merging**